server/controllers/storyController.js

```diff
@@ -0,0 +1,38 @@
+const storyModel = require('../models/storyModel');
+
+const getStory = (req, res) => {
+    storyModel.getStory(function (err, record) {
+        if (err) {
+            res.status(404).json(err.toString());
+            throw err;
+        } else {
+            res.status(200).json(record.rows);
+        }
+    });
+};
+
+const createStory = (req, res) => {
+    let newStory = new storyModel(req.body);
+    if (!newStory.storyID) {
+        res.status(400).send({ error:true, message: 'Please provide story ID.' });
+    } else if (!newStory.gridID) {
+        res.status(400).send({ error:true, message: 'Please provide grid ID.' });
+    } else if (!newStory.ts) {
+        res.status(400).send({ error:true, message: 'Please provide timestamp.' });
+    } else if (!newStory.text && !newStory.bucketName) {
+        res.status(400).send({ error:true, message: 'Please provide text or bucket name or both.' });
+    } else {
+        storyModel.createStory(newStory, function(err, record) {
```

Comment:
looks great
maybe in future we can have separate functions for 
1. validating request body properties
2. performing application logic
3. raising appropriate status code and sending response

Reply:
good point! i will look into that and separate those validation functions in the future. this one was solely for testing that the middleware was functioning as expected.